### ISSUE TITLE
"Post to timeline" block in page builder for the account.php.

### ISSUE THREAD

**Issue by andrewboon** [boonex/dolphin.pro#28]
Add a "post to timeline" block in page builder for the account.php page. When members login, that's the first page they are taken to. They view all the latest member posts, and post right from this page. Account.php is the most active/popular page on all my sites.

**Comment by AlexTr** [comment]
Actually it is aggregated list of notifications from friend, not a some sort of public place.

When user posts to the Timeline in own profile - it'll be displayed in own profile.
When user posts to the Timeline of another user - it'll be displayed in that profile Timeline.
When user posts to the Timeline on homepage - it'll be displayed in the public 

So, if user posts to the own private list of events in account - where "publicly" it will be displayed ?

**Comment by andrewboon** [comment]
We can show "own" updates to account.php timeline also. Point is that it's often the "Start" page for user, so Post to Timeline should be there. Quite typical for most social networks. Post goes to the user Timeline, but displays in Account feed/timeline as well.

**Comment by launch3** [comment]
Exactly. Account.php is a start page for users. Having used 3rd party EVO Wall (instead of current Timeline) on 3 sites has totally spoiled me and my users.

A typical Dolphin site has a small user base. Most users don't even make friends (or if they do, very few). This leaves the account.php "friends timeline" block essentially blank. It's basically a useless page! If users constantly see a blank page each time they login, they do not come back often.

On the other hand, having the account.php display a "post to timeline" block followed by "member timeline" block directly below that, shows ALL member activity in the community and encourages high participation rate. Account.php becomes a BIG hub of activity.

When user posts to the Timeline on homepage - choice: public / members only
When user posts to the Timeline on account page - choice: public / members only
When user posts to the Timeline in own profile - choice: own profile / members only / public

Admin Settings: admin should decide what posts are displayed by default in each location above. 
User Settings: userscan change to their own preferences in 'edit profile' page under 'edit profile privacy'.

With a small user base, these settings encourage growth.

As site grows and participation soars, settings can then filter from "member timeline" to "friends timeline" as desired on account.php.

Attached (image) is what account.php page can be. Great start page and hub!
![account-php-page](https://cloud.githubusercontent.com/assets/13964338/9469867/4af4f662-4b17-11e5-9ef5-5adf35972e5c.png)
